dist_tables/dml.rst

```diff
@@ -3,7 +3,7 @@
 Ingesting, Modifying Data (DML)
 ###############################
 
-The following code snippets use the distributed tables example dataset, see :ref:`ddl`.
+The following code snippets use the distributed tables example, see :ref:`ddl`.
```

Comment:
(Nitpick) After taking a fresh look at this, "distributed tables example" seems kind of vague. What about "...use the Github events schema defined in :ref:`ddl`"